### PR TITLE
Kochava Device Modes Doc

### PR DESCRIPTION
**Pull request by desusai7**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: destinations/kochava.md [hunk 1]
```diff
@@ -73,16 +166,125 @@ A sample `track` event for sending event data to Kochava looks like the followin
...
 According to the table above, this will change the `Product Added` event to `Add to Cart` event in Kochava dashboard and pass the properties along with this.
 
+## Screen
+
+The `screen` method allows you to record whenever a user sees the mobile screen, along with any associated optional properties. This call is similar to the `page` call, but is exclusive to your mobile device.
+
+A sample `screen` call looks like the following code snippet:
+
+```text
+MainApplication.rudderClient.screen("Tracking Page",
```
Comment (desusai7) [inline destinations/kochava.md:176]: Change the name of screen call to something relevant to screen.

File: destinations/kochava.md [hunk 2]
```diff
@@ -73,16 +166,125 @@ A sample `track` event for sending event data to Kochava looks like the followin
...
+            RudderProperty()
+                .putValue("url","www.kochava.com")
+                .putValue("referer","Rudderstack"))
+```
+
+In the above snippet, RudderStack captures all the information related to the screen being viewed, along with any additional info associated with that screen view event. In Kochava, the above `screen` call will be shown as - **"Screen View `Sample Screen Name` "** along with the properties.
```
Comment (desusai7) [inline destinations/kochava.md:182]: Update the `Sample Screen Name` with the screen Name you passed after making the changes said above

Comment (1abhishekpandey) [reply destinations/kochava.md]: Updated.

**Review by 1abhishekpandey — APPROVED** [review]

**Review by arnab-p — APPROVED** [review]

**Review by ameypv-rudder — APPROVED** [review]
@arnab-p @desusai7 made some changes to the content, but LGTM otherwise.

**Pull request merged**